src/pages/LandingPage/LandingPage.jsx

```diff
@@ -95,10 +95,11 @@ const LandingPage = () => {
             </div>
           </div>
         </div>
-
-        <div id="contact" className="min-h-screen py-12 px-6 text-white">
```

Comment:
`min-h-screen` means minimum-height = 1 screen height, you must re add this to get the missing space back.